.github/workflows/close-inactive.yaml

```
@@ -11,7 +11,7 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - uses: actions/stale@v6
+      - uses: actions/stale@v7.0.0
```

```suggestion
      - uses: actions/stale@v7
```
Set only major and will always run the latest